addons/stock/views/stock_picking_views.xml

```diff
@@ -126,6 +126,7 @@
             <field eval="12" name="priority"/>
             <field name="arch" type="xml">
                 <form string="Transfer">
+                    <field name="use_create_lots" invisible="1"/> <!-- used by embedded account.move view -->
```

Comment:
```suggestion
                    <field name="use_create_lots" invisible="1"/> <!-- used by embedded stock.move list view below -->
```
Little typo, doesn't have much to do with accounting. :smiley: 
Also I agree with nle, indenting the line just looks weird. Maybe you can move it after this line : https://github.com/odoo/odoo/blob/fe8e2845a16328824fa723fea7c2470e589f14fb/addons/stock/views/stock_picking_views.xml#L233

Reply:
thank you, updated

Reply:
Thanks, but don't forget about the second part. :wink:

Reply:
you're right! it is done now